.eslintrc.js

```diff
@@ -7,9 +7,11 @@ module.exports = {
   },
   plugins: [
     'jest', // We use the plugin jest so we can lint test files
+    'import',
```

Comment:
Giving the 'import/no-unresolved' rule added, is this necessary?

Reply:
By reading the documentation, it seems so.